_data/simple-icons.json

```diff
@@ -6962,6 +6962,21 @@
             "source": "https://github.com/openSUSE/artwork/blob/33e94aa76837c09f03d1712705949b71a246a53b/logos/buttons/button-colour.svg",
             "guidelines": "https://en.opensuse.org/Portal:Artwork"
         },
+        {
+            "title": "OpenTelemetry",
+            "hex": "000000",
+            "source": "https://cncf-branding.netlify.app/projects/opentelemetry/",
+            "guidelines": "https://cncf-branding.netlify.app/projects/opentelemetry/",
+            "license": {
+                "type": "CC-BY-4.0",
+                "url": "https://www.linuxfoundation.org/trademark-usage/"
+            },
+            "aliases": {
+                "aka": [
+                    "otel"
```

Comment:
Is this alias necessary? I can't find any OpenTelemetry related results in http://www.google.com/search?q=otel

Reply:
Yes, it's well known in the community: https://github.com/open-telemetry/opentelemetry-specification#acronym

Reply:
Good catch though, as I should have used `OTel`. I'll change it.